<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>主机:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>端口:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hostLine">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="portLine">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>信息.....</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>收到的信息:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendLine">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectBtn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>portLine</zorder>
  <zorder>hostLine</zorder>
  <zorder>messageLabel</zorder>
  <zorder>label_4</zorder>
  <zorder>sendLine</zorder>
  <zorder>sendBtn</zorder>
  <zorder>connectBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
